### DISCUSSION TITLE
honkit pdf can generate pdf, but all Chinese chars are in a mass

### DISCUSSION THREAD

**Discussion by mudongliang** [honkit/honkit#411] [Q&A]
After modifying the assembly to x86asm and buiding successfully, I found the generated pdf is in a mass for Chinese. What should I do to support?

P.S., I have prepared a branch named honkit to try honkit.

**Comment by azu** [comment]
please see https://honkit.netlify.app/config#pdf-options

```js
  "pdf": {
    "fontFamily": "<font name>"
  }
```

2024年5月5日(日) 18:01 Dongliang Mu ***@***.***>:

> After modifying the assembly to x86asm and buiding successfully, I found
> the generated pdf is in a mass for Chinese. What should I do to support?
>
> P.S., I have prepared a branch named honkit to try honkit.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/honkit/honkit/discussions/411>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAE2AWXWGDQEGTW3CMWWDTZAXYODAVCNFSM6AAAAABHHRR7IGVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGYYDQNZXGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
==================================
Name : azu
Mail   : ***@***.***
==================================

**Reply by mudongliang** [reply]
Our repo does not have a book.json. Can you please provide a book.json example? I can follow to create our own book.json.

**Comment by mudongliang** [comment]
I am a maintainer of linux-insides. Since legecy gitbook is already deprecated, I am looking for gitbook alternatives. Honkit is in our consideration. [siddhpant](https://github.com/siddhpant) submitted a [PR](https://github.com/0xAX/linux-insides/pull/837) to leverage honkit to generate pdf/epub/mobi releases.
